src/pages/SignUpPage/SignUpPage.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Container } from "@material-ui/core";
+import { SignUpForm } from "../../components/SignUp/SignUp";
+import { useStyles } from "./SignUpPage.style";
+
+const SignInPage = () => {
+  const classes = useStyles();
+  return (
+    <React.Fragment>
+      <Container maxWidth="md" className={classes.root}>
+        <SignUpForm />
+        <Link to="/signup" className={classes.link}>
+          Already have an account? Sign in
```

Comment:
It would be better if only 'Sign in' will be the link, wrap everything in Typography and make this text as Link

Reply:
Done